lib/resources/r4/encounter.yaml

```diff
@@ -547,6 +547,20 @@ fields:
         info_link: http://hl7.org/fhir/r4/v3/ObservationValue/cs.html
     note: Encounters created with meta.security set with code 'UNCERTREL' will be returned with an external encounter type.
 
+- name: Custom Attribute Extension
+  url: https://fhir.cerner.com/millennium/r4/encounters/encounter/#extensions
+  terminology_name: extension[x].extension[x].valueCodeableConcept
+  type: CodeableConcept
+  description: The value of the client defined custom attribute. This extension is nested under a <code>custom-attribute</code> extension.
+  action: terminology
+  binding:
+    description: The value of the client defined custom attribute. This binding is used for the <code>custom-attribute-value</code> extension nested under <code>customm-attribute</code> extensions.
+    note: The value of the custom attribute is not limited to a single codeSet.
+    terminology:
+      - display: Custom Attribute Value
+        system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/&lt;codeSet&gt;
+        info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes-and-systems/#proprietary-codes
+
```

Comment:
Shouldn't we list this extensions and also the two sub extensions as separate extensions. Similar to how Allocation is done for FinancialTransaction: https://github.com/cerner/fhir.cerner.com/blob/main/lib/resources/r4/financial_transaction.yaml#L218

Reply:
Hopped on a call with Keifer. We discussed the following:
- We don't need to list the two sub-extensions as separate extensions because we only support read/search on this extension presently.
- Update information about the CodeableConcept terminology binding found in this commit: https://github.com/cerner/fhir.cerner.com/pull/438/commits/f09e1c81f6b5cc0a015aad358333ac32609dbae5